### PR TITLE
Replaced references to users.User with get_user_model()

### PR DESCRIPTION
**Pull request by signalkraft**
While playing with an integration of the latest stable tag of Taiga into an existing Django application, I noticed that I couldn't set my own `AUTH_USER_MODEL` without breaking everything, due to hardcoded references to `users.User`. So I replaced it with `get_user_model()` and `settings.AUTH_USER_MODEL`.

I had to shuffle some `__init__` code around to get everything working without `AppRegistryNotReady` errors.

### CODE REVIEW

File: settings/testing.py [hunk 1]
```diff
@@ -29,6 +29,17 @@
...
+    'default': {
+        'ENGINE': 'transaction_hooks.backends.postgresql_psycopg2',
+        'NAME': 'test',
+        'USER': 'postgres',
+        'PASSWORD': 'postgres',
+        'HOST': 'localhost',
+        'PORT': '',
+    }
+}
+
```
Comment (bameda) [inline settings/testing.py]: In master we use Django 1.9 and this backend is not necessary https://docs.djangoproject.com/es/1.9/releases/1.9/#s-performing-actions-after-a-transaction-commit

**Comment by bameda** [comment]
I like the PR but could you fix this conflicts?

File: settings/testing.py (hunk 1 above)
Comment (signalkraft) [inline settings/testing.py]: That was a mistake, I was testing on 1.8 and forgot to remove it. I'll do that and fix the conflicts. Should I target stable or master?

Comment (bameda) [inline settings/testing.py]: master is better for include in the next release (early April)

**Comment by signalkraft** [comment]
I resolved the conflicts.

**Comment by bameda** [comment]
Great

File: taiga/timeline/api.py [hunk 2]
```diff
@@ -99,7 +100,7 @@ def retrieve(self, request, pk):
 
 
 class ProfileTimeline(TimelineViewSet):
-    content_type = "users.user"
+    content_type = settings.AUTH_USER_MODEL
     permission_classes = (permissions.UserTimelinePermission,)
```
Comment (bameda) [inline taiga/timeline/api.py]: tests are broken because:

```
settings.AUTH_USER_MODEL == "users.User" != "users.user"
```

**Comment by bameda** [comment]
Follow at https://github.com/taigaio/taiga-back/pull/679

**Pull request closed without merging**